pages/learn/develop/blocks.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Develop with blocks
+excerpt: A drop-in chunk of functionality that can be added to your application to provide.
+---
+
+# Getting Started with blocks
+
+A software package is a collection of individual files or resources that are packed together to provide a certain functionality as part of a larger system. Blocks are just that. They are drop-in, independent chunks of functionality built to handle a specific task, allowing you to focus on solving the hard problems instead.
```

Comment:
I added independent on my own as I felt some blocks could be used independently of balena. Although most of them currently use some balenaAPI or feature. I will drop that. 

I talked about software packages first, because I feel it generates a sense of familiarity to the concept and thus making the understanding of a new concept "seem" easier than it actually is. It also felt like a nice hook into starting the docs for blocks. I can remove/modify it if you strongly feel that way.

Reply:
@vipulgupta2048 I think we can drop this paragraph and consolidate those thoughts into the below one, as per my next comment. It flows better.